filters/diag/absorb.go

```diff
@@ -60,11 +61,18 @@ func NewAbsorbSilent() filters.Spec {
 	return withLogger(true, nil)
 }
 
-func (a absorb) Name() string                                            { return AbsorbName }
-func (a absorb) CreateFilter(args []interface{}) (filters.Filter, error) { return a, nil }
-func (a absorb) Response(filters.FilterContext)                          {}
+func (a *absorb) Name() string {
```

Comment:
why do we need pointers here? Not using pointers is a good indicator that the state of the filter doesn't change.

Reply:
I am following this suggestion https://dave.cheney.net/2016/03/19/should-methods-be-declared-on-t-or-t

Reply:
TBH, not too convincing in this particular case but I am fine with it :smile: